applications/ShapeOptimizationApplication/custom_utilities/mapping/mapper_vertex_morphing.h

```diff
@@ -114,8 +114,17 @@ class MapperVertexMorphing
         CreateListOfNodesOfDesignSurface();
         CreateFilterFunction();
         InitializeMappingVariables();
-        AssignMappingIds();        
-        ComputeMappingMatrix();
+        AssignMappingIds();
+
+        // optional flags
+        try {
+            mConsistentBackwardMapping = optimizationSettings["design_variables"]["consistent_backward_mapping"].GetBool();
+        }
+        catch (...)
+        {
+            mConsistentBackwardMapping = false;
+        }
```

Comment:
Thanks for the hint, (sorry for the confusing comment above, it was meant as a reminder for me to change it according to Riccardos suggestion)